entry_types/scrolled/package/src/entryState/sharing.js

```diff
@@ -72,26 +72,26 @@ export function useShareProviders(isPhone) {
   };
 
   return useMemo(() => {
-    return activeShareProviders(shareProviders, isPhone).map((provider) => {
+    return activeShareProviders(shareProviders, options.isPhonePlatform).map((provider) => {
       const config = sharing[provider];
       return ({
         name: config.name,
         icon: config.icon,
         url: config.url
       })
     })
-  }, [shareProviders, isPhone]);
+  }, [shareProviders, options]);
```

Comment:
Since the `options` object will be a new object on each render, this defeats the purpose of `useMemo`. You can instead destructure the options right in the parameter list (i.e. `useShareProviders({isPhonePlatform})`) and pass the boolean here as before

Reply:
Done